sonar-db/src/main/java/org/sonar/db/permission/PermissionMapper.java

```diff
@@ -30,6 +31,8 @@
 
   int countUsers(Map<String, Object> parameters);
 
+  List<UserWithPermissionDto> selectUsersByPermissionAndComponent(@Param("permission") String permission, @Param("componentId") long componentId, @Param("userIds") List<Long> userIds);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 184 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)